### PR TITLE
FaF performance tweak: updated dockerfile

### PR DESCRIPTION
**Pull request by errantmind**
Small tweak to test thread prioritization

### CODE REVIEW

File: frameworks/Rust/faf/faf.dockerfile [hunk 1]
```diff
@@ -13,4 +13,4 @@ RUN RUSTFLAGS="-Ctarget-cpu=native -Ztune-cpu=native -Zmutable-noalias=yes -Clin
    && strip ./target/x86_64-unknown-linux-gnu/release/faf-ex
 
 EXPOSE 8089
-CMD ./target/x86_64-unknown-linux-gnu/release/faf-ex
+CMD nice -19 ./target/x86_64-unknown-linux-gnu/release/faf-ex
```
Comment (joanhey) [inline frameworks/Rust/faf/faf.dockerfile:16]: Sorry,  Will you do that in your servers ??

Comment (errantmind) [reply frameworks/Rust/faf/faf.dockerfile:16]: Hey, I'm not sure what you mean. Do it where? In the code itself?

Comment (errantmind) [reply frameworks/Rust/faf/faf.dockerfile:16]: Nevermind, I'll move this into my code to simplify it

**Comment by joanhey** [comment]
I will be here again, for look your code in rust.

**Comment by joanhey** [comment]
A lot of people don't understand what is a benchmark.

It's a really cool app to change the code in our frameworks for be faster.
NOT a list of trips for look faster, that it if make the others fw will be faster too. But nobody will do it in his servers.
Thank you.

**Comment by joanhey** [comment]
And sorry but it's not your first time.
Read #6967

**Comment by errantmind** [comment]
> And sorry but it's not your first time. Read #6967

I don't see how changing the priority of the process is an issue. That is normal behavior and not a problem. It isn't about looking fast, it makes sense to ensure optimal scheduling for a critical piece of infrastructure is running. 

That said, I agree with the decision about routing mentioned in #6967. That optimization went too far but I will ask you not to use that to make assumptions about the rest of my code. I'm here to be the fastest but I will obey the rules. I want to make something interesting and useful.

**Pull request closed without merging**